content/zh/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -49,14 +49,6 @@ kubelet 会尝试通过 Kubernetes API 服务器为每个静态 Pod 自动创建
 
 {{% capture prerequisites %}}
 
-<!--
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
-
-This page assumes you're using {{< glossary_tooltip term_id="docker" >}} to run Pods,
-and that your nodes are running the Fedora operating system.
-Instructions for other distributions or Kubernetes installations may vary.
--->
-
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
```

Comment:
The correct way of fixing this is to move `<!--` after the `include` shortcode.

Reply:
great

Reply:
{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}	
\<!--	
This page assumes you're using {{< glossary_tooltip term_id="docker" >}} to run Pods,	
and that your nodes are running the Fedora operating system.	
Instructions for other distributions or Kubernetes installations may vary.	
\-->

Reply:
@tengqm @xichengliudui Thanks for your correction! I have repushed it.